remote_manager/src/nodes/remote_manager_node.cpp

```diff
@@ -0,0 +1,25 @@
+#include "remote_manager/remote_manager.h"
+
+int main(int argc, char **argv)
+{
+    ros::init(argc, argv, "remote_manager_node");
+    ros::NodeHandle nh;
+    ros::NodeHandle pn("~");
+
+    int frequency;
+    pn.param<int>("frequency", frequency, 20);
```

Comment:
ref: https://docs.ros.org/en/diamondback/api/rostime/html/classros_1_1Rate.html

```suggestion
    double frequency;
    pn.param<double>("frequency", frequency, 20.0);
```